pep-0001.txt

```diff
@@ -549,8 +562,9 @@ Once the PEP is ready for the repository, a PEP editor will:
 * Send email back to the PEP author with next steps (post to
   python-list & -dev).
 
-Updates to existing PEPs also come in to peps@python.org.  Many PEP
-authors are not Python committers yet, so PEP editors do the commits for them.
+Updates to existing PEPs should be submitted as a `GitHub pull request`_.
+Questions may of course still be sent to <peps@python.org>.  Many PEP authors
+are not Python committers yet, so PEP editors do the commits for them.
```

Comment:
We can remove the last sentence since we don't commit their updates for them anymore.